containers/python-bootstrap-container/entrypoint.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash 
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -e
+
+function save_to_temp_then_file() {
+    TEMP_FILE="$(echo mktemp)"
+    # Redirect output to temporary file TEMP_FILE
+    cat > TEMP_FILE
+    # Replace the original file
+    mv -f TEMP_FILE "${1}"
+}
+
+
+MONO_REPO_NAME="google-cloud-python"
+WORKSPACE_DIR="/workspace"
```

Comment:
Just curious: where does this `/workspace` directory live? (ie what file system? a top-level mount in Docker referencing someplace else?)

Reply:
Regretfully, there is not a lot of public documentation but googlers can find more information at `go/owlbot-bootstrapper`

The information is based on this section
```
The language-specific container will run in the workspace environment containing the following structure:

└── workspace/
      └── <mono-repo-that-was-cloned>/
```